pkg/checkpoint/manifest.go

```diff
@@ -78,10 +78,10 @@ func writeManifestIfDifferent(path, name string, data []byte) (bool, error) {
 		return false, nil
 	}
 	glog.Infof("Writing manifest for %q to %q", name, path)
-	return true, writeAndAtomicRename(path, data, 0644)
+	return true, writeAndAtomicRename(path, data, 0644, rootUID, rootGID)
 }
 
-func writeAndAtomicRename(path string, data []byte, perm os.FileMode) error {
+func writeAndAtomicRename(path string, data []byte, uid, gid int, perm os.FileMode) error {
```

Comment:
this func is busy enough i feel. probably should not couple chown into this function.

Reply:
I think it's better practice to force all callers to specify the UID and GID explicitly.